integration_tests/src/main/python/array_test.py

```diff
@@ -80,4 +81,15 @@ def test_orderby_array_of_structs(data_gen):
     assert_gpu_and_cpu_are_equal_sql(
         lambda spark : unary_op_df(spark, data_gen),
         'array_table',
-        'select array_table.a, array_table.a[0].child0 as first_val from array_table order by first_val')
\ No newline at end of file
+        'select array_table.a, array_table.a[0].child0 as first_val from array_table order by first_val')
+
+
+@pytest.mark.parametrize('data_gen', [byte_gen, short_gen, int_gen, long_gen,
+                                      FloatGen(no_nans=True), DoubleGen(no_nans=True),
+                                      string_gen, boolean_gen, date_gen, timestamp_gen], ids=idfn)
+def test_array_contains(data_gen):
+    arr_gen = ArrayGen(data_gen)
+    lit = gen_scalar(data_gen, force_no_nulls=True)
+    assert_gpu_and_cpu_are_equal_collect(lambda spark: debug_df(two_col_df(
+        spark, arr_gen, data_gen).select(array_contains(col('a'), lit.cast(data_gen.data_type)),
```

Comment:
Because we are doing random number generation, it is likely that all of these will come back as false.  If we want to do something like.

```
array_contains(col('a'), col('a')[5])
```

too that would be good.

Reply:
Added another `array_contains` clause similar to the original change.